packages/cli/test/models/StatusComparator.test.js

```diff
@@ -1,54 +1,55 @@
 'use strict'
 require('../setup')
 
-import { Contracts, App } from 'zos-lib'
+import { Contracts } from 'zos-lib'
 
 import push from '../../src/scripts/push'
-import linkStdlib from '../../src/scripts/link'
+import link from '../../src/scripts/link'
 import { bytecodeDigest } from '../../src/utils/contracts'
 import StatusChecker from '../../src/models/status/StatusChecker'
 import ZosPackageFile from '../../src/models/files/ZosPackageFile'
 import StatusComparator from '../../src/models/status/StatusComparator'
-import PackageAtVersion from '../../src/models/lib/PackageAtVersion';
 
 const ImplV1 = Contracts.getFromLocal('ImplV1')
 const AnotherImplV1 = Contracts.getFromLocal('AnotherImplV1')
 
-contract.skip('StatusComparator', function([_, owner, anotherAddress]) {
+contract('StatusComparator', function([_, owner, anotherAddress]) {
   const network = 'test'
   const txParams = { from: owner }
 
   describe('app', function () {
     beforeEach('initializing network file and status checker', async function () {
-      init.call(this, 'test/mocks/packages/package-empty.zos.json');
+      init.call(this, 'test/mocks/packages/package-empty.zos.json')
     })
-  
+
     beforeEach('deploying an app', async function () {
       await push({ network, txParams, networkFile: this.networkFile })
-      this.app = await App.fetch(this.networkFile.appAddress, txParams)
+      this.project = await this.checker.setProject()
+      this.directory = await this.project.getCurrentDirectory()
     })
 
-    testVersion();
```

Comment:
Do you have a linter that removes semicolons, or it's just OCD? :-P

Reply:
just OCD :joy: